allennlp/data/dataset_readers/semantic_parsing/grammar_based_text2sql.py

```diff
@@ -48,19 +54,31 @@ class GrammarBasedText2SqlDatasetReader(DatasetReader):
     """
     def __init__(self,
                  schema_path: str,
+                 database_file: str,
                  use_all_sql: bool = False,
                  remove_unneeded_aliases: bool = True,
+                 use_prelinked_entities: bool = True,
```

Comment:
I think most of the literature refers to this as "entity anonymization" so we might also want to name it that or "variable anonymization" or something like that for consistency.